service/tid/tid_darwin.go

```diff
@@ -0,0 +1,8 @@
+//go:build darwin
+// +build darwin
```

Comment:
`// +build darwin` (and other OS) was the annotation style pre-1.17 and is ignored by go starting with Go 1.18. I think we can remove these lines.